docs/.pages

```diff
@@ -1,9 +1,9 @@
 # customize the navigation menu
 # see https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
 nav:
+  - ...
```

Comment:
Nope, we don't want to do it that way as `...` is for items not cataloged here.

Home correspond to the README.md that we move to index.md at generation time. So we should instead just cite it:

```yaml
nav:
  - index.md
  - introduction
  - tutorials
  - how-to-guides
  - explanations
  - reference
  - ...